hooks/addon

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -eu
+vault="secret/$GENESIS_VAULT_PREFIX"
+target=$GENESIS_ENVIRONMENT
+
+
+list() {
+  echo "The following addons are defined:"
+  echo
+  echo "  list            List out all supported addons."
+  echo "  visit-prom      Visit the Prometheus Web UI (macOS only)"
+  echo "  visit-grafana   Visit the Grafana dashboard (macOS only)"
+  echo "  visit-alert     Visit the AlertManager dashboard (macOS only)"
+  echo
+}
+
+case $GENESIS_ADDON_SCRIPT in
+list)
+  list
+  exit 0
+  ;;
+
+visit-prom*)
+  if ! command -v open >/dev/null 2>&1; then
+    echo "The 'visit' addon script only works on macOS, currently."
```

Comment:
Fix the name of the addon hook in this error (and the other two, for other visit-* varieties)